src/content/guides/how-to-create-an-ethereum-account/index.md

```diff
@@ -49,20 +47,21 @@ Once you have saved your recovery phrase you should see your wallet dashboard wi
 
 ### Are my wallet and my Ethereum account the same?
 
-No, just like online banking you can have many different accounts all stored in one wallet application. Your 12 or 24 word phrase secures them all: it is like the seed of a big tree (which is why you must keep it safe). Each branch of the tree holds a key, and every key is one of your accounts. If you lose access to the wallet (i.e. your tree gets cut down), you can always restore all your different accounts using other software and the same seed (which will always grow the same tree).
+No. Each account has a unique Ethereum address and a secret key. The secret key is used to sign and send transactions. The wallet is a management tool that helps you to manage accounts. A single wallet might give access to several accounts, and a single account can be accessed by multiple wallets.
```

Comment:
this part can be confusing @jmcook1186 
1. we need to unify the focus/language and/or explain the difference between "key", "secret key" and a "secret recovery phrase". **Ideally use just one term.**

2.  I think the explanation fails short because it starts explaining the connection between secret key and address instead just saying that wallet only views an account and that there can be multiple accounts in one wallet.

Reply:
How about this? I think part of the technical issues on the  page stem from trying to use a single term for distinct concepts.

```suggestion
No. The wallet is a management tool that helps you to manage accounts. A single wallet might give access to several accounts, and a single account can be accessed by multiple wallets. Each account is really just a pair of keys. One key is used to create an address and can be shared freely. The other is a secret key used to sign and send transactions. The seed phrase is used to generate the secret key for an account (it is actually slightly more complicated because one seed phrase can generate multiple accounts). 
```

Reply:
```suggestion
No. The wallet is a management tool that helps you to manage accounts. A single wallet might give access to several accounts, and a single account can be accessed by multiple wallets. The seed phrase is used to create accounts that are then controlled by the wallet.
```

Reply:
Wondering if an ELI-5 analogy would help here... often when I explain a "seed" phrase, I draw an analogy to a tree that grows from an individual seed. This "seed" phrase can "grow" an entire tree of accounts... Each leaf is an account, with a corresponding key. Each unique seed will grow an entirely different tree of accounts.

This is pretty much why we use the word "seed" to begin with.... could help new users form mental models to something they already understand. 

---

Overall I'm fine with pretty much any of the above... feel free to ignore my analogy; just trying to think how we can convey what is needed simply with accurate terms.